src/Bricks/Model/BrickView.php

```diff
@@ -0,0 +1,15 @@
+<?php declare(strict_types=1);
+
+namespace Neusta\Pimcore\AreabrickConfigBundle\Bricks\Model;
+
+class BrickView
+{
+    public string $name;
+    public string $id;
+
+    /** @var array<mixed> */
+    public array $pageFullPaths;
+
+    /** @var array<string, mixed> */
```

Comment:
Here, too?

Reply:
I've tried with my last commit to introduce a typified version of the view model for Bricks.